src/components/Header/index.tsx

```diff
@@ -204,11 +205,33 @@ export default function Header({ title }: HeaderProps) {
     setSearchTags(newTags);
   };
 
+  const handleBack = async (e: React.MouseEvent<HTMLDivElement>) => {
+    e.preventDefault();
+    e.stopPropagation();
+    // Check if there's enough history to go back
+    if (window.history.length > 2) {
```

Comment:
`> 2` means if the new tab is created and then you nav to post, the history is `= 2` so the back doesn't work.